QueAns/admin.py

```diff
@@ -0,0 +1,76 @@
+from django.contrib import admin
+from .models import Question, Meetings, Message, Voting, Satisfaction
+# from datetime import date
+
+
+class MeetingsAdmin(admin.ModelAdmin):
+    list_display = ['meetings_text']
+
+    def get_readonly_fields(self, request, obj=None):
+        if request.user == obj.moderator:
+            return ['moderator']
+        else:
+            return ['id','meetings_text', 'venue', 'moderator', 'start_time', 'end_time']
+
+    def has_add_permission(self, request):
+        if request.user.is_superuser:
+            return True
+
+    def has_delete_permission(self, request, obj=None):
+        return False
+
+
+class QuestionAdmin(admin.ModelAdmin):
+    fields = ['question_text', 'anon_status', 'meeting_id','asker']
+    list_filter = ['meeting_id', ]
+    list_display = ['question_text']
+
+    def get_fieldsets(self, request, obj=None):
+        if obj is not None:
+            if obj.anon_status == 1:
+                return [(None, {'fields': ('question_text','meeting_id','anon_status',)})]
+
+        return [(None, {'fields': ('question_text', 'anon_status', 'meeting_id','asker')})]
+
+
+    def get_readonly_fields(self, request, obj=None):
+        if obj is not None:
+            if not request.user == obj.asker:
```

Comment:
Could be simply by `request.user != obj.asker`

Reply:
Just new to python :/